.github/workflows/policy-validate.yml

```
@@ -16,25 +16,23 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - name: Install pnpm
+      - name: Setup Node
         uses: pnpm/action-setup@v4
         with:
-          run_install: false
-
-      - name: Setup Node
+          version: 9.12.0
         uses: actions/setup-node@v4
```

YAMLlint (1.38.0)

[error] duplication of key "uses" in mapping

(key-duplicates)